kentico-michaelb/kontent-sample-app-conference-net
# Net core upgrade

### Motivation

Which issue does this fix? Fixes #30 

### How to test

Clean and Build without errors in Visual Studio.
Test navigation and links to ensure routing works appropriately.
